cfpb/cfgov-refresh
# Allow filtering of Enforcement Actions by Enforcement Status

- Adds status to the filterable list control and FilterForm
  - Updates statuses to be url friendly
  - Includes a script to update old statuses to the friendly format
  - adds status to the special-cased filterable_list util logic

Closes <jira>/INTAKE-3864

## Testing

- Pull & migrate
- Check the "Filter Enforcement Statuses" checkbox on the enforcement actions filter page in wagtail
- run the included `resave_enforcement` script with `./manage.py runscript resave_enforcement`
- note the inclusion of statuses in the filterable list control on the enforcement actions page
- Filtering should work as expected
- Statuses should still display as expected on any given enforcement action page

Once this is merged, we'll need to do the above steps in prod as well

<img width="946" alt="Screen Shot 2020-06-25 at 5 16 21 PM" src="https://user-images.githubusercontent.com/1558033/85807810-d9813880-b707-11ea-97f0-bd2be171b0be.png">
